src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -356,18 +356,8 @@ export default class DefaultDeviceController
 
     // Update device and stream
     this.chosenVideoTransformDevice = device;
-    const newMediaStream = this.activeDevices['video'].stream;
     this.logger.info('video transform device uses previous stream');
 
-    // Input is not a MediaStream. Update constraints
-    if (!(inner as MediaStream).id) {
-      const constraint = inner as MediaTrackConstraints;
-      constraint.width = constraint.width || this.videoInputQualitySettings.videoWidth;
-      constraint.height = constraint.height || this.videoInputQualitySettings.videoHeight;
-      constraint.frameRate = constraint.frameRate || this.videoInputQualitySettings.videoFrameRate;
-      await newMediaStream.getVideoTracks()[0].applyConstraints(constraint);
-    }
-
```

Comment:
Could you remind me what this change is for? I can't remember. The rest changes looks good to me.

Reply:
It's in the PR description:
>Delete the code path in DefaultDeviceController where we try to re-apply the video resolution and framerate when reusing the current video stream. That should not be needed. This is needed before in case the original input is a media track constraints and we always try to augment video resolution and frame rate.

Reply:
Oh, I forgot to check description. Thanks!